auditlog/models.py

```diff
@@ -208,6 +212,18 @@ def _get_pk_value(self, instance):
             pk = self._get_pk_value(pk)
         return pk
 
+    def _get_serialized_data_or_none(self, instance):
+        from .registry import auditlog
+
+        registry_opts = auditlog.get_model_fields(instance.__class__)
+        if not registry_opts["serialize_data"]:
+            return None
+
+        kwargs = registry_opts.get("serialize_kwargs", {})
+        kwargs.setdefault("fields", [x.name for x in instance._meta.fields])
```

Comment:
Part of the reason my team is searching for a replacement for our current auditing system is because we need to be able to NOT create new audit records when certain field change but we WANT them included in the object serialization. Consider the following example...

We have a model for `Person` and the state of each person instance is examined every _t_ minutes for changes that may require sending the state of the person to an external system. That is, we need to sync each person with an external system every time specific conditions are met and we examine those conditions every _t_ minutes. To track this, we store a value on the `Person` instance for `last_checked_for_sync` and `last_synced`. We do NOT need to bloat our database with audit logs for each examination when the only field that changes is incrementing `last_checked_for_sync` but we do want to have that information if we look back at historical logs and examine real changes. 

I think we have some options here though. Consider that in the `serialize_kwargs` one can already add an argument for `fields` and this will be passed to the native Django serializer. Therefore if one wants to limit the fields being serialized, he/she can specify them there. If that's not intuitive enough there is another option. We could add an argument in `serialize_kwargs` for choosing fields based on the the model field selections. Something like `use_auditlog_fields_only`? On the other hand, I quite like that the current `serialize_kwargs` are all passed directly to the Django serializer though and creating our own would betray that.

If you feel strongly that we need to only be serializing fields based on what the `exclude_fields` and `include_fields` shows, then I would be reluctantly okay with flipping the script and saying that if one really needs to override that behavior then he/she would need to specify all the model fields in the `serialize_kwargs` `fields` argument. It would just be a pain for me and others in my situation.

Reply:
As the new `serialized_data` is for saving the current state of the object, I would suggest including all fields and adding an option to limit it to auditlog field only. for example `use_auditlog_fields_only`.

Please remember we don't want to make it very complex and the main idea of the library is auditing

What do you think @alieh-rymasheuski

Reply:
Sounds good. I can add an option for `use_auditlog_fields_only` to be set in `serialize_kwargs`. I will do my best to keep it simple!

Reply:
@hramezani I've added an implementation of ``serialize_auditlog_fields_only``. Hopefully this works for everyone who needs that option. I wanted to keep the ``serialize_kwargs`` reserved for arguments that can be passed directly to the Django core serializer. I think its more discoverable and future proof that way but I'm open to nesting it within the ``serialize_kwargs`` if you feel differently.